schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE animals(
+    id int,
```

Comment:
- [OPTIONAL]
   - You can also set ID as primary key by `PRIMARY KEY (id)` and make it auto incremented with `id int NOT NULL IDENTITY(1,1)`